index.js

```diff
@@ -1,13 +1,11 @@
 var Mugshot = require('./lib/mugshot.js');
 
 Mugshot.adapters = {
-  WebdriverIO: require('./lib/adapters/webdriverio.js'),
-  LooksSame: require('./lib/adapters/looks-same.js')
+  WebdriverIO: require('./lib/adapters/webdriverio.js')
```

Comment:
Saw that it's being done in #60. You should update the PR name to indicate that we're only removing the differ adapter.